keel-api/src/main/kotlin/com/netflix/spinnaker/config/SwaggerConfiguration.kt

```diff
@@ -0,0 +1,62 @@
+package com.netflix.spinnaker.config
+
+import com.fasterxml.classmate.ResolvedType
+import com.fasterxml.classmate.TypeResolver
+import com.netflix.spinnaker.keel.api.ResourceSpec
+import com.netflix.spinnaker.keel.plugin.ResourceHandler
+import org.slf4j.LoggerFactory
+import org.springframework.beans.factory.config.YamlPropertiesFactoryBean
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.core.io.ClassPathResource
+import springfox.documentation.builders.PathSelectors
+import springfox.documentation.builders.RequestHandlerSelectors
+import springfox.documentation.spi.DocumentationType.SWAGGER_2
+import springfox.documentation.spring.web.plugins.Docket
+import springfox.documentation.swagger2.annotations.EnableSwagger2
+
+@Configuration
+@EnableSwagger2
+class SwaggerConfiguration {
+  @Bean
+  fun api(resolver: TypeResolver, handlers: List<ResourceHandler<*, *>>): Docket =
+    Docket(SWAGGER_2)
+      .apply {
+        handlers
+          .map { it.supportedKind.second }
```

Comment:
I think we should map by api version + supportedKind (so we don't end up with this issue again: https://github.com/spinnaker/keel/issues/527)